src/components/links/links.tsx

```diff
@@ -73,45 +72,3 @@ const ExternalLink = styled.a`
   text-decoration: none;
   color: inherit;
 `;
-
-const LinkButtonContainer = styled(Link)`
-  display: inline-block;
-  font-size: 16px;
-  font-weight: bold;
-  color: #fff;
-  text-decoration: none;
-  padding: 11px 16px;
-
-  background: linear-gradient(275.41deg, #543fd7 0%, #2756fd 100%);
-  border-radius: 30px;
-
-  /**
-   * todo: needs to be replaced with an svg. this also makes vertical align 
-   * middle possible / simpler   
-   */
-  svg {
-    transition: transform 0.2s ease-in;
-    margin-left: 20px;
-  }
-  :hover {
-    svg {
-      transform: translateX(2px);
-    }
-  }
-`;
-
-export const LinkButton = (props: LinkProps): JSX.Element => {
```

Comment:
good bye 👋 much better to deal only with a Link or Button and let the components pick the right semantic i.e. `button` or `a` internally👍